### PR TITLE
Separate animated card tests

### PR DESCRIPTION
**Pull request by msakowicz**
At this moment it is pointing to branch feature/ST-315-delete-animated-card-from-stjs-library https://github.com/SecureTrading/js-payments/tree/feature/ST-315-delete-animated-card-from-stjs-library
Before marge I have to change back to develop

Description: 
I separated animated card tests (for repo with only animated card) and place it in AnimatedCardTest.feature. I had to create new methods for animated card and input fields because in jspaymest-card repo there are no iframes and in jspayments are. Selenium requires additional action if iframes exist so I was no able reuse the same methods.

Concept is:
in jspayments-card repo set travis variable TEST_TAG as @animatedCardRepoTest  -> it will run only tests for animated card and fields validation (all without iframe)

in jspayments repo set travis variable TEST_TAG=@smokeTest or @fullTest or if animated card is visible: TEST_TAG=@smokeTest,@animatedCard (all in iframes)

### CODE REVIEW

File: src/test/java/resources/features/PaymentMethodsTests.feature [hunk 1]
```diff
@@ -98,26 +98,23 @@ Feature: Payment methods
...
-    Examples:
-      | cardNumber       | formattedCardNumber | expirationDate | cvc  | cardType |
-      | 340000000000611  | 3400 000000 00611   | 12/23          | 1234 | AMEX     |
-      | 4111110000000211 | 4111 1100 0000 0211 | 12/22          | 123  | VISA     |
     Examples:
-      | cardNumber          | formattedCardNumber    | expirationDate | cvc | cardType     |
+      | cardNumber       | formattedCardNumber | expirationDate | cvc  | cardType   |
+      | 340000000000611  | 3400 000000 00611   | 12/23          | 1234 | AMEX       |
+      | 4111110000000211 | 4111 1100 0000 0211 | 12/22          | 123  | VISA       |
+      | 6011000000000301 | 6011 0000 0000 0301 | 12/23          | 123  | DISCOVER   |
```
Comment (james-simpson-ST) [inline src/test/java/resources/features/PaymentMethodsTests.feature]: Now that these tests will exist on the animated card repo tests I think we can lower the number of examples on this repo down to only test AMEX and VISA

File: src/test/java/resources/features/PaymentMethodsTests.feature [hunk 2]
```diff
@@ -98,26 +98,23 @@ Feature: Payment methods
       | cardNumber       | expirationDate | cvc | cardType   |
       | 5100000000000511 | 12/22          | 123 | MASTERCARD |
 
-  @fullTest @animatedCard
+  @animatedCard
```
Comment (james-simpson-ST) [inline src/test/java/resources/features/PaymentMethodsTests.feature]: I think animatedCard tests should still be part of the fullTest but there should just be a need for less cases

File: src/test/java/resources/features/PaymentMethodsTests.feature (hunk 1 above)
Comment (msakowicz) [reply src/test/java/resources/features/PaymentMethodsTests.feature]: Unnecessary tests are already commented

File: src/test/java/resources/features/PaymentMethodsTests.feature (hunk 2 above)
Comment (msakowicz) [reply src/test/java/resources/features/PaymentMethodsTests.feature]: As we agreed on Teams we can connect two tags for @fullTest and @animatedCard if animated card will be set as visible on this repo

**Comment by msakowicz** [comment]
In last commit I've made some refactor to avoid unnecessary code duplication

**Pull request merged**